components/builder-scheduler/src/server/handlers.rs

```diff
@@ -50,11 +45,7 @@ pub fn schedule_get(req: &mut Envelope,
     let msg: proto::ScheduleGet = try!(req.parse_msg());
     println!("ScheduleGet: {:?}", msg);
 
-    let group_opt;
-    {
-        let ds = state.datastore().read().unwrap();
-        group_opt = ds.get_group(msg.get_group_id());
-    }
+    let group_opt = state.datastore().get_group(msg.get_group_id()).unwrap();
```

Comment:
This unwrap needs a comment about why its safe, and at the very least should be an expect.